web/components/application/hacker/pages/PageTwo/PageTwo.js

```diff
@@ -17,36 +19,159 @@ class PageTwo extends Page {
+        githubLink,
+        personalWebsiteLink,
+        linkedInLink,
+        resumeLink,
+        interestForNwHacks,
+        recentProject,
       },
     } = this.props;
 
     return (
       <HackerApplicationPageTemplate {...this.getPageTemplateProps()}>
-        <h1>About you <span role="img" aria-label="book">📚</span></h1>
+        <h2>Nice to meet you, {firstName}! <span role="img" aria-label="wavinghands">👋</span></h2>
         <p>
-          nwHacks is Western Canada’s largest collegiate hackathon taking place
-          on January 26th - 27th, 2019 at the University of British Columbia.
-          As expected, we receive a large number of applications every year.
-          We focus on curating a quality hackathon experience for each attendee.
-          For a fair assessment of your application,
-          we encourage you to put your best foot forward on this journey!<span role="img" aria-label="mountain">⛰️</span>
+          Help us get to know you better by providing as many links as you feel will support your application.
         </p>
-        <TextInput
-          label="Email"
-          name="hacker-application-email"
-          value={email}
-          onChange={(newEmail) => {
-            this.updateApplication({
-              email: newEmail,
-            });
-          }}
-          onBlur={() => this.setFieldAsBlurred('email')}
-          error={this.getErrorIfBlurred('email')}
+
+        <RadioGroup
+          name="first-hackathon"
+          label="Is this your first hackathon?"
+          className="dir-row margin-ends-giga"
+          onChange={(newIsFirstHackathon) => {
+            newIsFirstHackathon = newIsFirstHackathon === 'true';
+            this.updateApplication({ isFirstHackathon: newIsFirstHackathon });
+          }
+          }
+        >
+          <RadioButton
+            label="Yes"
+            value="true"
+            selected={isFirstHackathon === true}
+          />
+          <RadioButton
+            label="No"
+            value="false"
+            selected={isFirstHackathon === false}
+          />
+        </RadioGroup>
+
+        {/* // Todo - not exactly how you guys want to store the values of the check boxes in the database */}
+        <CheckboxGroup
+          name="interested-role"
+          label="How do you wish to contribute at nwHacks? Your choice will not affect your application and you can always change your mind."
+          className="dir-row margin-ends-giga"
+          onChange={(isCheckboxChecked) => {
+            isCheckboxChecked = isCheckboxChecked === 'true';
+            this.updateApplication({ isHacker: isCheckboxChecked });
+            this.updateApplication({ isDesigner: isCheckboxChecked });
+            this.updateApplication({ isHardware: isCheckboxChecked });
+            this.updateApplication({ isOther: isCheckboxChecked });
+          }
+          }
+        >
+          <Checkbox
+            label="Developer"
+            value="true"
```

Comment:
The value for all your checkboxes are the same. You need to make them unique per option.

One way to do this:

1. Create an object that maps the Checkbox value to a key in the application
```
const map = {
  hacker: "isHacker",
  designer: "isDesigner",
}
```

2. In each `Checkbox`, pass in the appropriate key from the map in the "value" prop
```
<Checkbox
  label="Developer"
  value="hacker"
  ...
```

3. In `onChange` callback function of `CheckboxGroup`, the argument you are getting is the "value" of the selected `Checkbox`. So all you have to do is:
`this.updateApplication({ map[selectedValue]: true })`

This doesn't handle "unselecting" a checkbox though, so you'd have to figure that out yourself for now. I'm not exactly sure what argument is supplied in the `onChange` function, maybe @chamkank has a better a better idea. Or you can also use the chrome debugger tool, and set a breakpoint in this callback function. If you don't know how to use the chrome debugger tool, just ping me and we can set up a call

Reply:
You could also give each checkbox its own `onChange` function instead of passing a single one down through the CheckboxGroup. This is probably the easiest approach. 

For example, the Developer checkbox would be:
```
<Checkbox
  label="Developer"
  value="true"
  className="margin-left-none"
  onChange={(e) => {
    this.updateApplication({ isDeveloper: e.target.checked });
  }}
/>
```